Source/Core/VideoCommon/Statistics.cpp

```diff
@@ -24,7 +25,8 @@ static Common::EventHook s_before_frame_event =
 static Common::EventHook s_after_frame_event = AfterFrameEvent::Register(
     [] {
       DolphinAnalytics::PerformanceSample perf_sample;
-      perf_sample.speed_ratio = SystemTimers::GetEstimatedEmulationPerformance();
+      perf_sample.speed_ratio =
+          Core::System::GetInstance().GetSystemTimers().GetEstimatedEmulationPerformance();
```

Comment:
woof, this is gonna be a gnarly one to move off the global instance.

Reply:
Yeah... the event hooking system in general I'm not really sure what to do with...